src/common-lib/distributions.ts

```diff
@@ -0,0 +1,88 @@
+import { FixedNumber } from 'ethers';
```

Comment:
can you add some tests for this file to verify that the calculations are correct? ideally a new dev would be able to look at the tests and quickly understand how to use these methods by seeing example pairs of input-output.

Reply:
updated with tests